data.sql

```diff
@@ -29,4 +29,69 @@ VALUES
     ('Squirtle', FALSE, 3, '1993-04-02', -12.13),
     ('Boarmon', TRUE, 7, '2005-06-07', 20.4),
     ('Blossom', FALSE, 3, '1998-10-13', 17),
-    ('Ditto', TRUE, 4, '2022-05-14', 22);
\ No newline at end of file
+    ('Ditto', TRUE, 4, '2022-05-14', 22);
+
+INSERT INTO
+    owners (full_name, age)
+VALUES
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittake', 38);
+
+INSERT INTO
+    species (name)
+VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+UPDATE
+    animals
+SET
+    species_id = 2
+where
+    name LIKE '%mon';
+
+UPDATE
+    animals
+SET
+    species_id = 1
+where
+    name NOT LIKE '%mon';
+
+UPDATE
+    animals
+SET
+    owner_id = 1
+where
+    name = 'Agumon';
+
+UPDATE
+    animals
```

Comment:
[OPTIONAL]
- [ ] It's very handy and time-consuming to access the id manually on different tables, It's better to use the name or whatever data you are trying to access by its name.
For example to assign a given  animal an owner, you my consider saying:
```sql
 UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
```